src/System.Threading/tests/Performance/Perf.EventWaitHandle.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.Xunit.Performance;
+
+namespace System.Threading.Tests
+{
+    public class Perf_EventWaitHandle
+    {
+        [Benchmark]
+        public void Set()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (EventWaitHandle are = new EventWaitHandle(false, EventResetMode.AutoReset))
+                using (iteration.StartMeasurement())
+                {
+                    are.Set(); are.Set(); are.Set();
+                    are.Set(); are.Set(); are.Set();
+                    are.Set(); are.Set(); are.Set();
```

Comment:
Consider changing this to be a SetReset test, such that all of the repeated operations do Set(); Reset(); rather than having the first one transition it from not-set to set and then all of the rest be nops in terms of changing the signaled state.